adb/adb_protocol.py

```diff
@@ -175,7 +175,6 @@ def __init__(self, header, data=''):
 
   def Write(self, usb, timeout_ms=None):
     """Send this message over USB."""
-    timeout_ms = usb.Timeout(timeout_ms)
```

Comment:
Removed call to usb.Timeout() here and in Read() because usb.BulkRead and usb.BulkWrite both already use self.Timeout() internally.